packages/api/src/apps/api/app.ts

```diff
@@ -147,10 +147,10 @@ export default async (env: ProcessEnv) => {
   await services.lspCreator.update();
   console.log("Got all LSP addresses");
 
-  await services.emps();
+  await services.emps(0);
   console.log("Updated EMP state");
 
-  await services.lsps.update();
+  await services.lsps.update(0);
```

Comment:
these changes dont have to do with the pairname change, but added this to start at block 0 for events on first update